activities/ActivityTemplate/VueJS/js/components/SugarL10n.js

```diff
@@ -75,6 +91,39 @@ Vue.component('sugar-localization', {
 			Object.keys(strings).forEach(function (key, index) {
 				strings[key] = vm.getString(key.substr(6));
 			});
+		},
+
+		// COnvert a UNIX timestamp to Sugarizer time elapsed string
+		timestampToElapsedString: function (timestamp, maxlevel, issmall) {
```

Comment:
It look like to an internal method. May be we could expose an easier method `SugarL10n.localizeTimestamp(timestamp)`?

Reply:
Yes we can create a simpler method. By the way, wouldn't it limit the functionality of this method? As we will have to pass default values for `maxlevel` and `issmall`. Maybe we can just add the default values to the parameters of this method itself?